.github/workflows/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+test.md
```

Comment:
- [ ] Kindly remove the `.gitignore` fom the workflows folder and place it in the root folder